wincent/liferay-portal
# LPS-111348 Archive Marketplace Themes

The **goal** of this PR is to archive what we call "Marketplace Themes". We started releasing these themes to the marketplace as a way to showcase the product capabilities:
- [Fjord](https://web.liferay.com/marketplace/-/mp/application/78803431)
- [Porygon](https://web.liferay.com/marketplace/-/mp/application/81920471)
- [Westeros Bank](https://web.liferay.com/marketplace/-/mp/application/78803695)

As of `7.3`, we've agreed to let the Experience Management teams decide how to better share these capabilities, so we're discontinuing these specific apps. 

I think the right initial approach is to move these apps to the `archived` folder... although we could also opt to simply delete the code...

**Steps to validate:**
- Code for the themes actually live within `modules/apps/archived`
- Build (`ant all`) runs properly and finishes successfully. The archived modules shouldn't appear in the logs
- When starting the server:
    - There are no "registered" messages in the log (`1 theme for fjord-theme is available for use
`)
    - The theme modules do not appear as active modules (`telnet localhost 11311 > lb theme`)
    - Navigating to `Control Panel > Sites > Sites >` the themes don't show as valid site templates (see screenshot)

<img width="1437" alt="Screen Shot 2020-04-02 at 11 48 45" src="https://user-images.githubusercontent.com/905006/78234947-3342df80-74d8-11ea-9842-89d315b495df.png">
